server/page-kit.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom/server';
+import { Shell } from '@financial-times/dotcom-ui-shell';
+import { Layout } from '@financial-times/dotcom-ui-layout';
+
+export default ({ response, next, shellProps, layoutProps }) => {
+        layoutProps.navigationData = response.locals.navigation;
+        layoutProps.headerOptions = { ...response.locals.anon};
```

Comment:
I think the scripts and stylesheets config could be abstracted here as well as it looks to be the same everywhere